.changes/unreleased/Dependencies-20230213-174442.yaml

```diff
@@ -3,4 +3,4 @@ body: Updates cryptography dependency to be <40.0.0
 time: 2023-02-13T17:44:42.716454Z
 custom:
   Author: surbias
-  Issue: "465"
```

Comment:
Could keep the issue number around to be consistent.

Reply:
None of the dependencies entries have issues. It's not consistent with the other three changed logs, but it's consistent with the other dependencies logs.